docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+version: '3.4'
+services:
+  web:
+    build:
+      context: ./resources/docker/web
+    ports:
+      - "8080:80"
+    volumes:
+      - .:/var/www/html
+    environment:
+      XDEBUG_CONFIG: "extension=xdebug.so remote_enable=1 remote_connect_back=1 remote_host=10.254.254.254 remote_port=9000 idekey=PHPSTORM remote_autostart=0"
+      DOCUMENT_ROOT: "/var/www/html/web"
+      PHP_XDEBUG_ENABLED: "1"
+      PHP_IDE_CONFIG: "serverName=Docker"
+      PHP_SENDMAIL_PATH: "/usr/sbin/sendmail -t -i -S mail:1025"
+
+  # This will compile the CSS once when docker-compose is started.
```

Comment:
We discussed it with @brummbar as well. We decided it to leave it like this for now and handle it in a follow up since this ticket has taken so long. The main thing is that I am not an expert in this and the frontend team is now using gulp to watch the folder actively. Also, we don't compile SASS generally so we can live with it being compiled on startup for now.

Reply:
OK let's make a followup and put a link to the ticket here.